orchestra/static/orchestra/common/components/website-iframe/website-iframe.directive.es6.js

```diff
@@ -32,6 +32,14 @@ export default function websiteIframe ($compile, $sce, $timeout) {
           iframe.setAttribute(attr, iframeAttrs[attr])
         }
 
+        var iframeWrappers = document.getElementsByClassName('iframe-wrapper')
+        for (var i = 0; i < iframeWrappers.length; i++) {
+          if (iframeWrappers[i].children[0].id === 'iframe-wrapper-' + $scope.id) {
```

Comment:
is there any way to push this sort of logic into a selector so we don't need the `if`?

if not, then approve!:)